jupyterhub/singleuser.py

```diff
@@ -134,6 +134,45 @@ def _confirm_exit(self):
         # disable the exit confirmation for background notebook processes
         ioloop.IOLoop.instance().stop()
     
+    def initialize(self, argv=None):
```

Comment:
I think we eventually want to define a simple extension API that can pass jupyterhub template content to the single user notebook template so this monkey patch isn't required.